dextra-components/src/components/Plot/src/Plot.ts

```diff
@@ -66,6 +77,8 @@ export class OslPlot extends OslData {
       grid: this.grid,
       color: {legend: this.colorLegend},
       inset,
+      x: {axis: this.x_axis, type: this.x_type}, 
+      y: {axis: this.y_axis, type: this.y_type}, 
```

Comment:
I think we should be more description here. y_axis and y_type are fairly open ended. If `y_axis` corresponds to `Plot.AxisOptions['anchor']` something like `yAxisLocation` would be more appropriate, if a bit verbose.

Additionally, we can make the typing here better. For example, in `y_axis` the only valid types are `"top" | "right" | "bottom" | "left";` in the Plot typing, unless I'm misunderstanding the way it is implemented in your approach. We may not need the converter here, if it is string or undefined, since we can have undefined with a fallback default eg (`...{axis: this.y_axis || 'left'...}`)

Reply:
I'm using the converter because I would like the value could be set to 'null', so that no x/y axis is showing. Previously in grouped bar chart, `x_axis="null"`. If I change it to : 
```
@property({ type: String })
  xAxisAnchor?: "bottom" | "top" | "left" | "right" = "bottom"
```

Is there a way of setting its value to null?

Reply:
in [21aca46](https://github.com/open-spatial-lab/dextra/pull/4/commits/21aca468057fb1f2d8a798507653fba84b1df7bd): 
```
@property({ converter: interpretFuncJsonOrString })
  xAxisAnchor?: Plot.AxisOptions["anchor"] | null = "bottom";
```

Reply:
Gotcha, the converter is appropriate here then!